models/user.go

```diff
@@ -15,21 +15,15 @@ type User struct {
 func GetUser(id int64) (User, error) {
 	u := User{}
 	err := db.Where("id=?", id).First(&u).Error
-	if err != nil {
-		return u, err
-	}
-	return u, nil
+	return u, err
```

Comment:
Ah, yes, this particular code "smell" is probably all over the Gophish codebase. It really was just a preference at the time, since I just didn't feel right returning whatever error I was getting from gorm. Having the return nil made it feel a bit safer to me, even though functionally it didn't make a difference. And I agree that this is cleaner. 👍

Reply:
To me it seems a bit redundant, but if you feel it is safer, I'm happy to remove some commits and just leave the test cases! So please let me know what do you think! 😄

Reply:
Nah, no worries. I agree completely that this is a cleaner look.